src/main/java/online/book/store/repository/order/OrderRepository.java

```diff
@@ -0,0 +1,11 @@
+package online.book.store.repository.order;
+
+import java.util.Set;
+import online.book.store.model.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    Set<Order> findAllByUserId(Long userId);
```

Comment:
```suggestion
    List<Order> findAllByUserId(Long userId, Pageable pageable);
```
don't forget about `Pageable`